src/database/models/students.model.js

```diff
@@ -1,25 +1,23 @@
-const { Schema } = require('mongoose');
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
 const connection = require('../index');
 
 const StudentSchema = new Schema({
-    studentId:{
-        type: String,
-        unique: true,
+  studentId: {
+    type: String,
+    unique: true,
+  },
+  firstName: String,
+  lastName: String,
+  email: String,
+  phone: String,
+  program: String,
+  courses: [
+    {
+      type: mongoose.Types.ObjectId,
+      ref: 'Course',
     },
```

Comment:
You want a single source of truth. Either keep this, or keep the `students` field in the `courses` document. But not both